src/components/ContactUsForm/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Col, Row } from "antd";
+import { RiSendPlaneFill } from "react-icons/ri";
+import "./index.css";
+
+const ContactUsForm = () => {
+  const [t] = useTranslation();
+  const [formValue, setFormValue] = useState({
+    name: "",
+    email: "",
+    message: "",
+  });
+
+  // The change in isSubmitted state triggers submission message
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  // Converts object to proper form data(string)
+  const encode = (data) => {
```

Comment:
since this is a "pure function" (can google it) that is independent from the component i'd say it actually makes more sense to move this below the imports (in the same file), but outside the component. and call it "encodeFormData"

Reply:
separating functions like that makes it clear to the reader that it has no dependency on anything inside the component.

Reply:
Thanks for reviewing and explanation. I moved the function to the new file. PTAL